cmd/otelcontribcol/Dockerfile

```diff
@@ -1,8 +1,15 @@
+FROM golang:1.13.8 as golang
```

Comment:
Can we change `as golang` into `as builder`? I think it shows better the image purpose.

Reply:
sure, one sec 😹